main.go

```diff
@@ -15,279 +15,17 @@
 package main
 
 import (
-	"bytes"
-	"encoding/json"
-	"flag"
-	"fmt"
-	"io/ioutil"
-	"net/http"
-	"os"
-	"strings"
-
-	conf "github.com/fairwindsops/polaris/pkg/config"
-	"github.com/fairwindsops/polaris/pkg/dashboard"
-	"github.com/fairwindsops/polaris/pkg/kube"
-	"github.com/fairwindsops/polaris/pkg/validator"
-	fwebhook "github.com/fairwindsops/polaris/pkg/webhook"
-	"github.com/sirupsen/logrus"
-	apitypes "k8s.io/apimachinery/pkg/types"
-	_ "k8s.io/client-go/plugin/pkg/client/auth" // Required for other auth providers like GKE.
-	"sigs.k8s.io/controller-runtime/pkg/client/config"
-	"sigs.k8s.io/controller-runtime/pkg/manager"
-	"sigs.k8s.io/controller-runtime/pkg/runtime/signals"
-	"sigs.k8s.io/controller-runtime/pkg/webhook"
-	"sigs.k8s.io/yaml"
+	"github.com/fairwindsops/polaris/cmd/polaris"
 )
 
 const (
 	// Version represents the current release version of Polaris
 	Version = "0.6.0"
+	// TODO make sure this gets set.
+	commit = "n/a"
```

Comment:
What's this for?

Reply:
Copied from Goldilocks and not needed and can be safely removed

Reply:
Well, it was used for the version output, maybe it still would be valuable

Reply:
I'd say let's get rid of it. There's no way to check the commit hash in without changing the commit hash, so I can't imagine what this is meant for.